repos/spack_repo/builtin/packages/parsec/package.py

```diff
@@ -38,6 +37,8 @@ class Parsec(CMakePackage, CudaPackage):
         url="https://github.com/ICLDisco/parsec/archive/refs/tags/v1.1.0.tar.gz",
     )
 
+    patch("apply-header.patch", when="@4.0.2411")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
@alalazo I've added the comments.  The patch was developed by me for a future release of parsec: https://github.com/ICLDisco/parsec/pull/750